odoo/fields.py

```diff
@@ -1812,7 +1813,13 @@ def convert_to_column(self, value, record, values=None, validate=True):
         # Detect if the binary content is an SVG for restricting its upload
         # only to system users.
         if value[:1] == b'P':  # Fast detection of first 6 bits of '<' (0x3C)
-            decoded_value = base64.b64decode(value)
+            try:
+                # validate = True : to decode only base64 encoded values.
+                # translate(None, delete=b'\r\n') : to allow the decoding of values with retour lines.
+                # For exemple, binary files in emails.
```

Comment:
I'd move that to the commit message, possibly extended to clarify that by default base64.b64decode will just strip anythign which isn't part of the base64 alphabet from the input, then it'll try to decode *that*, so it's possible to feed it complete garbage and get different complete garbage instead of the error one could have expected.